src/OSPSuite.Presentation/Extensions/CurveChartExtensions.cs

```diff
@@ -28,10 +28,14 @@ public static Color SelectNewColor(this CurveChart chart)
          return colors[newIndex];
       }
 
-      public static void UpdateCurveColorAndStyle(this CurveChart chart, Curve curve, DataColumn dataColumn, IReadOnlyCollection<DataColumn> dataColumns)
+      public static void UpdateCurveColorAndStyle(this CurveChart chart, Curve curve, DataColumn dataColumn, IReadOnlyCollection<DataColumn> dataColumns, bool isLinkedDataToSimulation = false)
       {
          // Finds color from a related column
-         if (dataColumnContainsRelatedColumns(dataColumn))
+
+         if (isLinkedDataToSimulation)
+            curve.Color = chart.Curves.FirstOrDefault()?.Color ?? chart.SelectNewColor();
```

Comment:
why first or default>? should not that be the color of the linked sthg, whatever this something is